Prepare environment for "brand translation"

leonampd opened the pull request:
Change package name on composer.json, create an environment to run the module with docker. Also add a simple e2e test to test if the environment can setup a magento ecommerce. This is the first step to allow us to translate the module.

leonampd committed:
composer: change package name

change package name on composer. It permits install via composer.

leonampd committed:
environment: add docker-compose

Includes a `.env` file and a docker-compose to permit run magento and
pagarme module

leonampd committed:
wip

leonampd committed:
environment: applies changes on env variables and docker-compose

The alexcheg/magento2 image was changed: the author added new required
variables. So it add the environment variables and change selenium and
firefox versions to run behat on container as you can see [here](https://github.com/pagarme/docker-magento2/pull/6)

leonampd committed:
tests: add magento e2e test

Add a simple e2e test to validate that magento is running and showing
products on homepage

lflimeira commented on Test/Acceptance/MagentoBaseContext.php:
```diff
@@ -0,0 +1,60 @@
+        }
+    }
+
+    public function getPage()
```
What do you think about isolate this method?

leonampd committed:
acceptance: implement a trait for manipulate pages

Create a new trait for pages manipulation. It has, for now, a method to
retrieve a page from a valid session

lflimeira commented on Test/Acceptance/Page.php:
```diff
@@ -0,0 +1,9 @@
+<?php
+
+trait Page
```
It could be more generic than we could use for others functionalities. Does it makes sense?

lflimeira reviewed (approved):
LGTM!!!
